modules/oe_whitelabel_search/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenEuropa Whitelabel Search
+
+This module offers a searchbox block handled by a plugin.
+
+# Requirements
+
+The following modules must be installed and enabled to use the Search Box block:
```

Comment:
rather a option then a requirement, the view and autocomplete parts are optional